web/views/command_version_view.ex

```diff
@@ -1,6 +1,8 @@
 defmodule Cog.V1.CommandVersionView do
   use Cog.Web, :view
 
+  alias Cog.Commands.Help.CommandFormatter
```

Comment:
Do you mean something like `Cog.V1.CommandVersionView.CommandFormatter`? Feels weird to put that in a place that's so api route specific.